atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/OracleDbTableFactory.java

```diff
@@ -44,11 +45,27 @@ public DbMetadataTable createMetadata(String tableName, ConnectionSupplier conns
     }
 
     @Override
-    public DbDdlTable createDdl(String tableName, ConnectionSupplier conns) {
+    public DbDdlTable createDdl(TableReference tableName, ConnectionSupplier conns) {
         return new OracleDdlTable(tableName, conns, config);
     }
 
     @Override
+    public DbTableInitializer createInitializer(ConnectionSupplier conns) {
+        /* No initialization required for Oracle */
+        return new DbTableInitializer() {
+            @Override
+            public void createUtilityTables() {
+                //no op
+            }
+
+            @Override
+            public void createMetadataTable(String metadataTableName) {
+                //no op - will be required for Oracle to work
```

Comment:
Should probably do basic verification check that utility & metadata table exists.